apps/dashboard/app/views/layouts/nav/_group.html.erb

```diff
@@ -1,27 +1,26 @@
-<li class="dropdown" title="<%= group.title %>" >
-  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= group.title %><span class="caret"></span></a>
-  <ul class="dropdown-menu">
+<li class="nav-item dropdown" title="<%= group.title %>" >
+  <a class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= group.title %><span class="caret"></span></a>
+  <div class="dropdown-menu">
```

Comment:
I think we should keep structure of elements throughout these navbars for accessibility.  Seems like the only we need to change are the css classes.

Reply:
So to clarify @johrstrom, are you saying whatI I'm thinking that we should continue to use ul like before since this is actually a list.